UI/data/locale/en-US.ini

```diff
@@ -1277,8 +1277,10 @@ About.Info="OBS Studio is a free and open source video recording and live stream
 About.Donate="Make a Contribution"
 About.GetInvolved="Get Involved"
 About.Authors="Authors"
+About.Patrons="Patrons"
 About.License="License"
 About.Contribute="Support the OBS Project"
+About.Copyright="© 2012 - 2022 OBS Project"
```

Comment:
Might be worth using a %1 variable and using Qt to dynamically insert the current year, rather than depending on translators.

Reply:
I actually did experiment with this and wasn't sure if having the date be dependent on the user environment was desirable. Though in hindsight my concerns were probably misplaced, and **QDate::currentDate().year()** should make this trivially easy to implement.